CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 # Changelog
 Changelog for NeoFS Node
 
+## [Unreleased]
+
+## [0.22.0] - 2021-06-29 - Muuido (무의도, 舞衣島)
+
+Storage nodes with a group of network endpoints.
+
+### Added
+- Support of Neo wallet credentials in CLI ([#610](https://github.com/nspcc-dev/neofs-node/issues/610)).
+- More reliable approval of trust value by IR ([#500](https://github.com/nspcc-dev/neofs-node/issues/500)).
+- Storage node's ability to announce and serve on multiple network addresses ([#607](https://github.com/nspcc-dev/neofs-node/issues/607)).
+- Validation of network addresses of netmap candidates in IR ([#557](https://github.com/nspcc-dev/neofs-node/issues/557)).
+
+
+### Fixed
+- Approval of objects with with duplicate attribute keys or empty values ([#633](https://github.com/nspcc-dev/neofs-node/issues/633)). 
+- Approval of containers with with duplicate attribute keys or empty values ([#634](https://github.com/nspcc-dev/neofs-node/issues/634)). 
+
+### Changed
```

Comment:
- IR does not include nodes with LOCODE derived attributes to the network map (412)
- IR uses `morph/client` packages for contract invocations (496)
- Writecache decreases local size when objects are flushed (568)
- IR can override global configuration values only in debug build (363)